### PR TITLE
Pass test without inst ids

### PR DESCRIPTION
**Pull request by drmingdrmer**
# Description

### fix protobuf ops to pass test without instIDs.
- Use uniform field index.

- Rename Message.Request to Message.TypeRequest and so is
    Message.TypeReply, for the upcoming Request and Reply message.

- Comment out several fields in struct Replica.

## Type of change

- **Refactoring**
- **Test changes**

# Checklist:

- [ ] **Style**:       My code follows the **style guidelines** of this project
- [x] **Self-review**: I have performed a **self-review** of my own code
- [ ] **Comment**:     I have **commented my code**, particularly in hard-to-understand areas
- [ ] **Doc**:         I have made corresponding changes to the **documentation**
- [ ] **No-warnings**: My changes generate **no new warnings**
- [ ] **Add-test**:    I have added **tests** that prove my fix is effective or that my feature works
- [x] **Pass**:        New and existing **unit tests pass** locally with my changes
- [ ] **Dep**:         Any **dependent** changes have been merged and published in downstream modules

### CODE REVIEW

File: components/epaxos/src/data/protos/instance.proto [hunk 1]
```diff
@@ -24,19 +24,22 @@ message BallotNum {
...
 message Instance {
 
+    BallotNum last_ballot  = 11;
+    BallotNum ballot       = 12;
+    InstanceID instance_id = 13;
+    repeated Command cmds  = 21;
+
     // initial_deps is the dependent instance ids when an instance is created.
-    repeated InstanceID initial_deps                = 1;
+    repeated InstanceID initial_deps                = 31;
```
Comment (wenbobuaa) [inline components/epaxos/src/data/protos/instance.proto]: 改成这样的2位数有什么特殊含义吗

File: components/epaxos/src/data/protos/instance.proto [hunk 2]
```diff
@@ -24,19 +24,22 @@ message BallotNum {
 // Instance is the internal representation of a client request.
 message Instance {
 
+    BallotNum last_ballot  = 11;
+    BallotNum ballot       = 12;
```
Comment (lishulong) [inline components/epaxos/src/data/protos/instance.proto]: 每个message里面的field的index不重复就行了吧

File: components/epaxos/src/data/protos/instance.proto (hunk 1 above)
Comment (drmingdrmer) [reply components/epaxos/src/data/protos/instance.proto]: 没啥特殊意义, 预留一些空间,以后增加字段时如果要保持兼容也不会太乱, field出现的顺序跟数字的顺序能大概保持一致.

File: components/epaxos/src/data/protos/instance.proto (hunk 2 above)
Comment (drmingdrmer) [reply components/epaxos/src/data/protos/instance.proto]: 是的.

预留一些空间,以后增加字段时如果要保持兼容也不会太乱, field出现的顺序跟数字的顺序能大概保持一致.

全局保持一致以后复制粘贴啥的可能有点方便.

**Pull request merged**